src/common/telemetry/src/logging.rs

```diff
@@ -228,8 +239,15 @@ pub fn init_global_logging(
 
         // Configure the error file logging layer with rolling policy.
         let err_file_logging_layer = if !opts.dir.is_empty() {
-            let rolling_appender =
-                RollingFileAppender::new(Rotation::HOURLY, &opts.dir, "greptimedb-err");
+            let rolling_appender = RollingFileAppender::builder()
+                .rotation(Rotation::HOURLY)
+                .filename_prefix("greptimedb-err")
+                // Simply set maximum to constant 30 here
+                .max_log_files(30)
```

Comment:
Why not keep it the same with other logs? I think we should keep them consistent.

Reply:
There won't be too much error logs. But it's ok to use the same option.

Reply:
Will do.